SI/Sem.04/Pract.04/Task15.cpp

```diff
@@ -0,0 +1,38 @@
+#include <iostream>
+
+bool isPrime(int n) {
+	if (n == 1) {
+		return false;
+	}
+	for (int i = 2; i <= sqrt(n); i++) {
+		if (n % i == 0) {
+			return false;
+		}
+	}
+	return true;
+}
+
+void factorise(int n) {
+	if (isPrime(n)) {
+		std::cout << "number is prime." << std::endl;
+		return;
+	}
+	std::cout << n << " = ";
+	for (int i = 2; i <= n; i++) {
```

Comment:
You should loop to the square root of the number
Also you can optimize it a bit for the even numbers